react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -30,6 +30,15 @@ export default class AbstractPageReloadOverlay extends Component<*, *> {
      * @static
      */
     static propTypes = {
+        /**
+         * The details.
+         * NOTE: Used by PageReloadOverlay only.
```

Comment:
this is the AbstractPageReloadOverlay, so IMHO this comment is not needed. However, I'd like to see what "The details" is explained a bit more thoroughly :-)